tempesta_fw/t/functional/helpers/tf_cfg.py

```diff
@@ -0,0 +1,142 @@
+import configparser
+import os
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class TestFrameworkCfg():
+
+    def __init__(self):
+        self.defaults()
+        config_path = os.path.dirname(os.path.realpath(__file__ + '/..'))
+        self.config.read(config_path + '/tests_config.ini')
+
+    def defaults(self):
+        self.config = configparser.SafeConfigParser()
+        self.config.read_dict({'General': {'verbose': '0',
+                                           'duration': '10',
+                                           'arp': 'True',
+                                           'concurent_connections': '-1'},
+                               'Client': {'ip': '127.0.0.1',
+                                          'mac': 'ff:ff:ff:ff:ff:ff',
+                                          'hostname': 'localhost',
+                                          'user': 'root',
+                                          'port': '22',
+                                          'ab': 'ab',
+                                          'wrk': 'wrk',
+                                          'siege': 'siege',
+                                          'workdir': '/root/client',},
+                               'Tempesta': {'ip': '127.0.0.1',
+                                            'mac': 'ff:ff:ff:ff:ff:ff',
+                                            'hostname': 'localhost',
+                                            'user': 'root',
+                                            'port': '22',
+                                            'dir': '/root/tempesta',},
+                               'Server': {'ip': '127.0.0.1',
+                                          'mac': 'ff:ff:ff:ff:ff:ff',
+                                          'hostname': 'localhost',
+                                          'user': 'root',
+                                          'port': '22',
+                                          'nginx': 'nginx',
+                                          'workdir': '/root/nginx',
+                                          'resourses': '/srv/http/'}
```

Comment:
It's better to use `/var/www/html/` as a default directory for Apache HTTPD or Nginx